test/testdata/namer/yield.rb.symbol-table-raw.exp

```diff
@@ -3,14 +3,14 @@ class <C <U <root>>> < <C <U Object>> () @ (... removed core rbi locs ..., Loc {
     method <S <C <U <root>>> $1><N <U <static-init>> $CENSORED> (<blk>) @ Loc {file=test/testdata/namer/yield.rb start=2:1 end=27:14}
       argument <blk><block> @ Loc {file=test/testdata/namer/yield.rb start=??? end=???}
   class <C <U Main>> < <C <U Object>> () @ Loc {file=test/testdata/namer/yield.rb start=2:1 end=2:11}
-    method <C <U Main>><U blockpass> (&blk) @ Loc {file=test/testdata/namer/yield.rb start=7:5 end=7:24}
-      argument &blk<block> @ Loc {file=test/testdata/namer/yield.rb start=7:19 end=7:23}
+    method <C <U Main>><U blockpass> (blk) @ Loc {file=test/testdata/namer/yield.rb start=7:5 end=7:24}
+      argument blk<block> @ Loc {file=test/testdata/namer/yield.rb start=7:19 end=7:23}
     method <C <U Main>><U blockyield> (<blk>) @ Loc {file=test/testdata/namer/yield.rb start=15:5 end=15:19}
       argument <blk><block> @ Loc {file=test/testdata/namer/yield.rb start=??? end=???}
     method <C <U Main>><U main> (<blk>) @ Loc {file=test/testdata/namer/yield.rb start=19:5 end=19:13}
       argument <blk><block> @ Loc {file=test/testdata/namer/yield.rb start=??? end=???}
-    method <C <U Main>><U mixed> (&blk) @ Loc {file=test/testdata/namer/yield.rb start=11:5 end=11:20}
```

Comment:
can we update pretty printing code to keep `&`?

Reply:
I think this is actually inconsistent with everything else right now:

```
  class <C <U Object>> < <C <U BasicObject>> (<C <U Kernel>>) @ Loc {file=https://github.com/sorbet/sorbet/tree/master/rbi/core/object.rbi start=removed end=removed}
    method <C <U Object>><U bar> (b, d, &e) @ Loc {file=jez.rb start=2:1 end=2:21}
      argument b<keyword> @ Loc {file=jez.rb start=2:9 end=2:11}
      argument d<keyword, repeated> @ Loc {file=jez.rb start=2:13 end=2:16}
      argument &e<block> @ Loc {file=jez.rb start=2:18 end=2:20}
    method <C <U Object>><U foo> (a, c, &e) @ Loc {file=jez.rb start=1:1 end=1:19}
      argument a<> @ Loc {file=jez.rb start=1:9 end=1:10}
      argument c<repeated> @ Loc {file=jez.rb start=1:13 end=1:14}
      argument &e<block> @ Loc {file=jez.rb start=1:16 end=1:18}
```

i.e., we don't print `*` nor `:` nor `**`, but we do print `&` for block args.

and you can already see the arginfo attributes because they're printed on the lines right below.

also, printing that list at all seems like a product of when there was a separation between the symbols and the things that were in the arguments list, which doesn't exist anymore. maybe we should just stop printing that list?

Reply:
Ack, I'm onboard.